features/bootstrap/Context.php

```diff
@@ -152,4 +153,18 @@
             return false;
         }
     }
+
+    /**
+     * @Given /^that Science Digests are enabled$/
+     */
+    public function thatScienceDigestsAreEnabled()
+    {
+        if ($this->getSession()->getDriver() instanceof KernelDriver) {
+            putenv('FEATURE_DIGEST_CHANNEL=true');
```

Comment:
as for the previous `putenv` hiding the condition behind something like `isJavaScript()` helps with maintenance